test/FunctionsV2/TimerTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace WebJobs.Extensions.DurableTask.Tests.V2
```

Comment:
I realized we could theoretically support this test for Functions V1, we would just have to write some reflection based code in PlatformSpecificHelpers.FunctionsV1. Not sure if that is worth the effort though.

Reply:
Probably not worth backporting to V1. I'm thinking we should try to limit new features to V2 and just backport important bug fixes to V1.